packages/next/src/server/app-render/create-error-handler.tsx

```diff
@@ -139,11 +139,12 @@ export function createHTMLErrorHandler(
   dev: boolean,
   isNextExport: boolean,
   reactServerErrors: Map<string, DigestedError>,
-  allCapturedError: Array<unknown>,
+  allCapturedErrors: Array<unknown>,
   silenceLogger: boolean,
-  onHTMLRenderError: (err: any) => void
+  onHTMLRenderError: (err: any, isRSCError: boolean) => void
```

Comment:
Why are we not just calling this for RSC errors in createHTMLReactServerErrorHandler?

Then we can omit reporting the error in the SSR phase. Is there some reason why we don't have access to the error context there?